firmware/controllers/trigger/trigger_central.cpp

```diff
@@ -216,6 +217,12 @@ void TriggerCentral::resetCounters() {
 	triggerState.resetRunningCounters();
 }
 
+void TriggerCentral::resetAccumSignalData() {
+	memset(lastSignalTimes, 0xff, sizeof(lastSignalTimes));	// = -1
+	memset(accumSignalPeriods, 0, sizeof(accumSignalPeriods));
+	memset(accumSignalPrevPeriods, 0, sizeof(accumSignalPrevPeriods));
+}
```

Comment:
See vars definition in https://github.com/rusefi/rusefi/pull/592/commits/d13171d5988fc832be348e84bf721507024b2d48